src/middleware/featureFlag.ts

```diff
@@ -4,6 +4,16 @@ import logger from "@root/logger/logger"
 import { RequestHandlerWithGrowthbook } from "@root/types"
 import { getNewGrowthbookInstance } from "@root/utils/growthbook-utils"
 
+// Keep one GrowthBook instance at module level
+// The instance will handle internal cache refreshes via a SSE connection
+const gb = getNewGrowthbookInstance(config.get("growthbook.clientKey"))
+
+gb.loadFeatures({ autoRefresh: true }).catch((e: unknown) => {
+  logger.error(
```

Comment:
should we just pass the `error` directly instead of doing `JSON.stringify`?

Reply:
Good catch and yes we should!

If `e` is an `Error` here, `json.stringify()` will show us nothing at all. Better leave the logger to extract and format the info as needed.

I had copied the line from the middleware handling below to reduce the changeset to a minimum, but indeed we can make the logging much better.

I'll mark this PR as a dependent of the logger PR, so we can merge it later.